utils/src/main/java/com/nedap/archie/datetime/DateTimeParsers.java

```diff
@@ -27,13 +27,25 @@ public static TemporalAccessor parseDateTimeValue(String text) {
                 //Not parseable as a standard public object from datetime. We do not implement our own yet (we could!)
                 //so fallback to the Parsed object. The Parsed object is package-private, so cannot be added as a reference
                 //to the parseBest query, unfortunately.
-                return DateTimeFormatters.ISO_8601_DATE_TIME.parse(text);
+                return DateTimeFormatters.ISO_8601_DATE_TIME.parse(text, LocalDate::from);
```

Comment:
The format in the try catch seems the same as this one. Could this try/catch be replaced with the following with the same result?

```
                return DateTimeFormatters.ISO_8601_DATE_TIME.parseBest(text, LocalDate::from, Year::from);
```

Reply:
@pieterbos this has been a trial  and error process and this nested try {} catch {} finally ended to work.

Reply:
I'll create a branch and try later today, then show you the results - my suggestion should have the exact same behaviour and the tests should pass without modification, parseBest is equivalent to the try/catch here, but in less code.

Reply:
I created a pull request on your branch that does this at https://github.com/ehrbase/archie-fork/pull/1 :)